common/src/verify.rs

```diff
@@ -123,19 +123,26 @@ pub struct CommitSequenceVerifier {
     phase: Phase,
     reserved_state: ReservedState,
     commits: Vec<Commit>,
+    total_commits: Vec<Commit>,
 }
 
 impl CommitSequenceVerifier {
     /// Creates agenda new `CommitSequenceVerifier` with the given block header.
     pub fn new(start_header: BlockHeader, reserved_state: ReservedState) -> Result<Self, Error> {
         Ok(Self {
-            header: start_header,
+            header: start_header.clone(),
             phase: Phase::Block,
             reserved_state,
-            commits: vec![],
+            commits: Vec::new(),
```

Comment:
Is there any difference between `vec![]` and `Vec::new()`?

Reply:
In my knowledge, `vec!` macro will be expanded to `Vec::new()` on compilation. The point is not to use both of them in same code.

Reply:
No big difference.